spec/yabeda/gauge_spec.rb

```diff
@@ -24,4 +24,38 @@
     set_gauge
     expect(adapter).to have_received(:perform_gauge_set!).with(gauge, built_tags, metric_value)
   end
+
+  describe "#increment" do
+    context "when gauge has no initial value" do
+      before { gauge.increment(tags) }
+
+      it { expect(gauge.get(tags)).to eq 1 }
```

Comment:
IMO correct way to test it would be checking that adapters have received this value:
```suggestion
      it  { gauge.increment(tags) expect(adapter).to have_received(:perform_gauge_set!).with(gauge, tags, 1) }
```